_projects/13-build-tools.md

```diff
@@ -8,9 +8,23 @@ status: Active
 type: project
 active: true
 home: false
-description: "Performance improvements, new features and simplifications in sbt"
+description: "Contributions to Zinc, Scala's incremental compiler, and sbt."
 ---
 
-[Zinc Improvements](https://github.com/sbt/zinc/labels/Scala%20Center)
+This initiative focuses on improving day-to-day tools that all Scala developers depend on: [Zinc](https://github.com/sbt/zinc) and [sbt](https://github.com/sbt/sbt).
 
-[Improve builds with direct dependencies](https://github.com/scalacenter/advisoryboard/blob/master/proposals/009-improve-direct-dependency-experience.md)
\ No newline at end of file
+Our involvement in these projects have helped them to improve on the following
+topics:
+1. Reproducibility.
+2. Performance.
+3. Usability.
+4. Documentation.
+5. Welcoming experience to OSS developers.
+
+To see our contributions to Zinc [follow this link](https://github.com/sbt/zinc/labels/Scala%20Center).
+  
+To see our contributions to sbt [read this thread](https://contributors.scala-lang.org/t/reply-to-your-feedback-on-sbt/89://contributors.scala-lang.org/t/reply-to-your-feedback-on-sbt/892).
```

Comment:
You mean the bullet list? The bullet list doesn't list anything concrete at all.